tests/e2e/test_args.sh

```diff
@@ -0,0 +1,12 @@
+cd $GOPATH/src/k8s.io/kubernetes
```

Comment:
Please add a check of KUBECONFIG at the beginning. If not set, report error and exit

Reply:
okk Sir

Reply:
How should we access the KUBECONFIG?